app/src/main/java/ca/marshallasch/veil/FragmentSignUp.java

```diff
@@ -54,6 +55,20 @@ public FragmentSignUp(){
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState)
     {
         View view = inflater.inflate(R.layout.fragment_signup, container, false);
+        ((MainActivity) getActivity()).getSupportActionBar().hide();
+
+        // navigate back to the landing screen
+        view.setFocusableInTouchMode(true);
+        view.requestFocus();
+        view.setOnKeyListener((v, keyCode, event) -> {
+            if (event.getAction() == KeyEvent.ACTION_DOWN) {
```

Comment:
I think you might have to override the default onClick().

Reply:
This works, its the same block of code that you had in the dashboard fragment, but I moved it so that you can go right to the dashboard without having to go through the login screen first